include/constants/battle.h

```diff
@@ -61,8 +61,8 @@
 #define BATTLE_TYPE_REGI             0x4000
 #define BATTLE_TYPE_GHOST            0x8000
 #define BATTLE_TYPE_POKEDUDE         0x10000
-#define BATTLE_TYPE_PALACE           0x20000
-#define BATTLE_TYPE_ARENA            0x40000
+#define BATTLE_TYPE_PALACE           0x20000 // TODO: Update name
+#define BATTLE_TYPE_ARENA            0x40000 // TOOD: Update name
```

Comment:
1) are they really used in-game? 
2) typo

Reply:
Yes, the names were just copied from Emerald.